.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+# Reference doc: https://circleci.com/docs/2.0/project-walkthrough/
```

Comment:
Thanks for adding this. If you thought that nobody would read it, guess what: I am reading it.

Reply:
Well actually by following this I was able to understand everything that is happening in this config.

Reply:
Glad it helped.